install.sh

```diff
@@ -205,6 +205,12 @@ update () {
     git pull
     cd ..
 
+    echo -e "\nUpdating Supervisor SIGHTS extension"
+    cd supervisor_sights_config
+    git pull
+    cd ..
+    python3 -m pip install ./supervisor_sights_config
```

Comment:
Good thinking. We don't need the `--upgrade` flag do we?

Reply:
It didn't seem to complain when I used it without the upgrade flat, and it successfully installed 0.0.2 over 0.0.1 anyway.

Reply:
Too easy. If we don't update the version number, that flag will force install it I think. Shouldn't be a problem if we update the version number whenever we make a change though.